example-playbooks/restore_scylla_manager_backup/restore.yaml

```diff
@@ -135,6 +154,16 @@
           \g<1> - seeds: {{ old_seeds }}
         backrefs: yes
 
+    - name: Restart Scylla service
```

Comment:
I think we don't need this one since we are going to do a restart to load system_schema tables.